src/foam/nanos/actioncommand/ActionCommand.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.actioncommand',
+  name: 'ActionCommand',
+
+  documentation: 'Command object containing an object and pending action to be done on it',
+
+  properties: [
+    {
+      class: 'FObjectProperty',
+      name: 'object',
```

Comment:
Should this be the whole object or just its 'id'? I thought we would just send the id.